runtime/lua/vim/lsp/protocol.lua

```diff
@@ -1182,6 +1182,90 @@ protocol.Methods = {
   workspace_willRenameFiles = 'workspace/willRenameFiles',
   --- The `workspace/workspaceFolders` is sent from the server to the client to fetch the open workspace folders.
   workspace_workspaceFolders = 'workspace/workspaceFolders',
+  --- The `workspace/didChangeWorkspaceFolders` notification is sent from the client to the server when the workspace
+  --- folder configuration changes.
+  workspace_didChangeWorkspaceFolders = 'workspace/didChangeWorkspaceFolders',
```

Comment:
this is not sorted. workspace_didChangeWorkspaceFolders should go before workspace_workspaceFolders , etc...